tests/unit/type/zset.tcl

```diff
@@ -1760,12 +1782,12 @@ start_server {tags {"zset"}} {
             # equal or greater than set size.
             foreach size {10 20} {
                 set res [r zrandmember myzset $size]
-                assert_equal [llength $res] 10
+                assert_equal [llength $res] $card
                 assert_equal [lsort $res] [lsort [dict keys $mydict]]
 
                 # again with WITHSCORES
                 set res [r zrandmember myzset $size withscores]
-                assert_equal [llength $res] 20
+                assert_equal [llength $res] [expr {$card * 2}]
```

Comment:
we are missing a call to `check_member_and_score` here (not verifying PATH2).

Reply:
Yes it has been added

Reply:
for consistency, you can add check_member to the block above too.

Reply:
done